app/views/home/_nav.html.erb

```diff
@@ -0,0 +1,19 @@
+<nav class='py-2 container-fluid px-5 d-flex justify-content-between align-items-center bg-dark user gap-8 bs w-100'>
+      <ul class="d-flex justify-content-between align-items-center w-50 mb-0">
+        <li><%= link_to 'Public recipe', root_path %></li>  
+      </ul>
+      <% if user_signed_in? %>
+      <div class='d-flex align-items-center gap-1 text-white '>
+        <svg style='width: 32px' fill="none" stroke="currentColor" stroke-width="1.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+          <path stroke-linecap="round" stroke-linejoin="round" d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z"></path>
+        </svg>
+        <span><%= current_user.name %></span>
+      </div>
+      <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "custom-button btn btn-info btn-sm" %>
+      <% else %>
+        <div class='d-flex align-items-center gap-3 text-white '>
+          <%= link_to 'Sign in', new_user_session_path , class: "custom-button btn btn-info btn-sm" %>
+          <%= link_to 'Sign up', new_user_registration_path , class: "custom-button btn btn-info btn-sm" %>
+        </div>
+      <% end %>
+    </nav>
```

Comment:
- [ ] it would be better to add all the pages that users can see like foods recipes in order to complete the requirements 
<img width="686" alt="Screenshot 2023-02-10 at 17 42 30" src="https://user-images.githubusercontent.com/61790587/218150148-9d254269-3798-47f6-92b7-2e14826fe59e.png">

Reply:
For this one somehow it wasn't practical to add all rendered pages on the nav bar the fact that foods and recipes are associated with a particular user. The login, registration, logout, and public recipes page did fit well with that requirement

Reply:
@devMunyi yes you should add food and recipes to the nav bar

Reply:
Okay done